Autopilot/Path Manager/PathManager.c

```diff
@@ -340,7 +340,23 @@ char followWaypoints(PathData* currentWaypoint, float* position, float heading,
 
             *sp_Heading = (int)followStraightPath((float*)&waypointDirection, (float*)targetCoordinates, (float*)position, heading);
         }
-        else{
+        else{            
+            
+            // if target waypoint is a hold waypoint the plane will follow the orbit until the waypoint is updated or removed
+            if (targetWaypoint->type == HOLD_WAYPOINT) {
```

Comment:
Have you thought about how to exit the hold? I would think the best way to do it is to have an exitHold command, that would move on to the next waypoint if the plane was in a hold.